projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -47,6 +47,7 @@ <h2 [innerHtml]="product.nameHtml"></h2>
         {{ 'productDetails.noReviews' | cxTranslate }}
       </div>
       <div
+        *ngIf="!product.multidimensional"
```

Comment:
`query:` could it be the case that customers can already have multidimensional products in their stores despite the lack of SPA support? If so (but I'm guessing it's unlikely) this `ngIf` statement could be a breaking change.

Reply:
I don't see this as a breaking change. We cannot guarantee the stability of the application, if customers use unsupported features.
Consider a scenario where a customer adds a third-party library, and after updating SPA, it stops working. Would that be a breaking change?

Reply:
Good point! Nevertheless, compared to the usage of third-party lib, we provide a new statement to the template which will change how DOM looks in some conditions. Note that the `multidimensional` field has been part of the `Product` interface for 5 years. I guess nothing blocked users from basing their custom logic on it. That said, I believe it is a breaking change in some corner cases. The question is whether we should care and handle it with a toggle. I don't have a strong opinion on this.

Reply:
Without this condition, the multi-d product will have both product prices. I've considered many different approaches and the above is the cleanest one.